webrtc-charter.html

```diff
@@ -209,14 +214,6 @@ <h3>
 
           <p>The Working Group will deliver specifications that cover at least the following functions, unless they are found to be fully specified within other Working Groups' finished results:</p>
           <dl>
-            <dt>Media Stream Functions</dt>
-            <dd>API functions to manipulate media streams for interactive real-time communications, connecting various processing functions to each other, and to media devices and network connections, including media manipulation functions for e.g. allowing to synchronize streams. Supplementary functions such as recording of media streams are also in scope.</dd>
-            <dt>Audio Stream Functions</dt>
-            <dd>An extension of the Media Stream Functions to process audio streams, to enable features such as automatic gain control, mute functions and echo cancellation.</dd>
-            <dt>Video Stream Functions</dt>
-            <dd>An extension of the Media Stream Functions to process video streams, to enable features such as bandwidth limiting, image manipulation or "video mute".</dd>
-            <dt>Functional Component Functions</dt>
-            <dd>API functions that allow to query for the components present in an implementation, instantiate them, and connect them to media streams.</dd>
             <dt>Data Access Functions</dt>
```

Comment:
I think Data Access Functions for media streams belong with the Media functions.
We also have data access functions that belong in the comms part ("encoded data"), so perhaps both places?